src/components/StatusStrip/StatusStrip.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect, ReactNode } from "react";
+
+import "./_status-strip.scss";
+
+type Status = {
+  label: string;
+  count: number;
+};
+
+type Props = {
+  statusList: {
+    model: Status[];
+  };
+};
+
+export default function StatusStrip({ statusList }: Props) {
+  type Status = {
+    groupName: string;
+    statement: ReactNode;
+    count: number;
+  };
+  const [status, setStatus] = useState<Status>({
```

Comment:
I'm not sure you need all this logic any longer and can just loop through and render the data from the props if my new type suggestion above is ok.